### PR TITLE
Logging chart

### PR DESCRIPTION
**Pull request by joshmeranda**
**Problem:**
Implement the logging feature as described in [577](https://github.com/harvester/harvester/issues/577).

**Solution:**
Installer the `rancher-logging` and `rancher-logging-crd` charts as `ManagedCharts` by default and route logs to local loki deployment using the logging operator from BonzaiCloud.

**Related Issue:**
https://github.com/harvester/harvester/issues/577
https://github.com/harvester/harvester/issues/2644
https://github.com/harvester/harvester/issues/2645
https://github.com/harvester/harvester/issues/2646
https://github.com/harvester/harvester/issues/2647

**Test plan:**

cc @guangbochen @bk201 @w13915984028

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ steps:
     dockerfile: package/Dockerfile.webhook
     password:
       from_secret: docker_password
-    repo: "rancher/harvester-webhook"
+    repo: "joshmeranda/harvester-webhook"
```
Comment (bk201) [inline .drone.yml]: @joshmeranda I guess we can keep all repos with existing values. And don't worry that the PR needs to be tested with another installer PR, reviewers can handle this.

File: deploy/charts/harvester-journal/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```
Comment (bk201) [inline deploy/charts/harvester-journal/Chart.yaml]: This looks like a Harvester-specific chart. The convention is to create the chart in the https://github.com/harvester/charts repo and publish it. Then we can use it as a dependency chart in the `harvester/harvester` repo. You can check [other charts](https://github.com/harvester/charts/tree/master/charts) as an example. 

cc @guangbochen

File: .drone.yml (hunk 1 above)
Comment (bk201) [reply .drone.yml]: And similar changes in other files.

File: scripts/build-iso [hunk 3]
```diff
@@ -7,9 +7,10 @@ cd $(dirname $0)/..
 
 echo "Start building ISO image"
 
-HARVESTER_INSTALLER_VERSION=master
+HARVESTER_INSTALLER_VERSION=logging
```
Comment (bk201) [inline scripts/build-iso]: ditto, please remove these changes. We can keep the original values for
- build-iso
- ci
- test-integration

File: deploy/charts/harvester-journal/Chart.yaml (hunk 2 above)
Comment (joshmeranda) [reply deploy/charts/harvester-journal/Chart.yaml]: created PR for harvester-charts: https://github.com/harvester/charts/pull/94/

File: scripts/build-iso (hunk 3 above)
Comment (joshmeranda) [reply scripts/build-iso]: reversed changes to those files

File: .drone.yml (hunk 1 above)
Comment (joshmeranda) [reply .drone.yml]: reversed changes to those files

**Comment by w13915984028** [comment]
Replaced by https://github.com/harvester/harvester-installer/pull/333 , close.

**Pull request closed without merging**